### PR TITLE
[Excel] (Custom functions) Various fixes to broken links in Excel CFs docset

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[55530ec](https://github.com/OfficeDev/office-js-docs-pr/pull/1095/commits/55530ecf67d51dc50ec1bfd6a669c53350b1794b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixes/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixes/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixes/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906171927293425-1095%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FakTso5D5blA0dR%2FHjkQ2Y30CZ0O7i0mMCODJ81p8aU%3D&st=2019-06-17T19%3A23%3A28Z&se=2019-07-18T19%3A28%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9bc491a](https://github.com/OfficeDev/office-js-docs-pr/pull/1095/commits/9bc491a2f9aa3dcc728f9ef1ffb1050f7e4bbaf6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixes/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixes/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-linkfixes/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906171928314028-1095%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LmCMMDC1hJEWF4M%2BOntDSFfYzwxrHBJ1mDH6W%2BJASN0%3D&st=2019-06-17T19%3A24%3A30Z&se=2019-07-18T19%3A29%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-naming.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ A function's `name` shouldn't be named the same as:
 
 - Any cells between A1 to XFD1048576 or any cells between R1C1 to R1048576C16384.
 
-- Any Excel 4.0 Macro Function (such as `RUN`, `ECHO`).  For a full list of these functions, see [this article](https://www.microsoft.com/en-us/download/details.aspx?id=1465).
+- Any Excel 4.0 Macro Function (such as `RUN`, `ECHO`).  For a full list of these functions, see [this pdf](https://d13ot9o61jdzpp.cloudfront.net/files/Excel%204.0%20Macro%20Functions%20Reference.pdf).
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-naming.md]:
```suggestion
- Any Excel 4.0 Macro Function (such as `RUN`, `ECHO`).  For a full list of these functions, see the [Excel Macro Functions Reference](https://d13ot9o61jdzpp.cloudfront.net/files/Excel%204.0%20Macro%20Functions%20Reference.pdf).
```

File: docs/excel/custom-functions-json-autogeneration.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ To denote a custom function parameter as optional, do one of the following:
 * Use an optional parameter. For example: `function f(text?: string)`
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-json-autogeneration.md]: maybe update the ms.date

**Pull request merged**